fiftyone/core/collections.py

```
@@ -3292,8 +3292,19 @@ def _set_labels(self, field_name, sample_ids, label_docs, progress=False):
 
-    def _delete_labels(self, ids, fields=None):
-        self._dataset.delete_labels(ids=ids, fields=fields)
+    def _delete_labels(self, labels, fields=None):
+        self._dataset._delete_labels(labels, fields=fields)
+
+    def _map_values(self, in_values, in_field, *out_fields):
+        view = self.select_by(in_field, in_values)
+        _in_values, *_all_out_values = view.values([in_field, *out_fields])
+
+        results = []
+        for out_field, _out_values in zip(out_fields, _all_out_values):
```

Ruff (0.11.9)

Loop control variable `out_field` not used within loop body

Rename unused `out_field` to `_out_field`

(B007)